src/ui/components/SecondaryToolbox/ReactDevTools.tsx

```diff
@@ -355,6 +357,12 @@ class ReplayWall implements Wall {
           res.returned.object,
           "canOverflow"
         );
+        const sources = await sourcesByIdCache.readAsync(this.replayClient);
+
+        if (componentFunctionPreview.preview?.functionLocation) {
+          updateMappedLocation(sources, componentFunctionPreview.preview.functionLocation);
```

Comment:
There are other places where `ObjectPreview.functionLocation` is used and they will run into the same problem. I think `ObjectPreview.functionLocation` should be updated directly in `preCacheObject()`.